test/replutil.jl

```diff
@@ -281,9 +281,9 @@ let undefvar
     err_str = @except_str 0::7 TypeError
     @test err_str == "TypeError: typeassert: expected Type, got $Int"
     err_str = @except_str "" <: AbstractString TypeError
-    @test err_str == "TypeError: issubtype: expected Type, got String"
+    @test err_str == "TypeError: <:: expected Type, got String"
```

Comment:
`TypeError: in <:, expected Type, got String` maybe? there aren't any function names that end in a comma, are there?